app/views/schools/add_participants/confirm.html.erb

```diff
@@ -0,0 +1,65 @@
+<% content_for :title, "Confirm details" %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: back_link_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Confirm these details</h1>
+
+    <p class="govuk-body">We will send this person an email to sign in and add any further details.</p>
+
+    <dl class="govuk-summary-list govuk-!-margin-bottom-9">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Details
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <div><%= add_participant_form.full_name %></div>
+          <div><%= add_participant_form.email %></div>
+        </dd>
+        <dd class="govuk-summary-list__actions">
+          <%= govuk_link_to({ action: :show, step: "details" }) do %>
+            Change <span class="govuk-visually-hidden">personal details</span>
+          <% end %>
+        </dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Type
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <%= add_participant_form.type_options[add_participant_form.type.to_sym] %>
+        </dd>
+        <dd class="govuk-summary-list__actions">
+          <%= govuk_link_to({ action: :show, step: "type" }) do %>
+            Change <span class="govuk-visually-hidden">participant type</span>
+          <% end %>
+        </dd>
+      </div>
+
+      <%# <% if @add_participant_form.type == "ect" %1> %>
+      <%#   <div class="govuk-summary-list__row"> %>
+      <%#     <dt class="govuk-summary-list__key"> %>
+      <%#       Mentor %>
+      <%#     </dt> %>
+      <%#     <dd class="govuk-summary-list__value"> %>
+      <%#       <% if @mentor %1> %>
+      <%#         <%= @mentor.full_name %1> %>
+      <%#       <% else %1> %>
+      <%#         Not yet assigned %>
+      <%#       <% end %1> %>
+      <%#     </dd> %>
+      <%#     <dd class="govuk-summary-list__actions"> %>
+      <%#       <%= govuk_link_to({ action: :show, step: "mentor" }) do %1> %>
+      <%#         Change <span class="govuk-visually-hidden">mentor</span> %>
+      <%#       <% end %1> %>
+      <%#     </dd> %>
+      <%#   </div> %>
+      <%# <% end %1> %>
```

Comment:
Somehow I don't think it should be here 😂

Reply:
it should it's just waiting for the mentor changes stan is making! at the moment all the variables it's referring to are undefined

Reply:
@kicferk1 - Yup, this is getting a bit odd to have multiple PRs for a single journey. Not really sure what the best solution is - maybe viewing a single PR by a well defined commit history?

Reply:
Fine, let's add the remaining part of the feature here.

I know dev retro happened after a lot of this code was written. Buut, in future I would expect to see a PR adding this feature in as simple a way as possible, and then, once that goes in, another PR that adds an abstraction of multi-page form. Cause this is exactly the thing that makes reviewing things harder.

Reply:
I think the problem lies in the definition of `as simple a way as possible`. I don't think writing this in a RailsWay(tm) would make it simple at all. There would be a massive amount of repetitions - not only around all the separate action but also around the `back` link options which would require writing conditions for both going back and forward depending on the state (not to mention a number of edge cases when the conditions are time-sensitive). The form flow would be heavily obfuscated by the amount of boilerplate code and, quite likely, it would take more time to write it this way to handle all the odd cases.

So I'd say our approach is pretty much the same, but we differ in definition of `as simple as possible` - I think this is pretty simple - there is no metaprogramming involved, no custom DSL, the whole logic is split in two files and views. Refactor is definitely pending with some extra edge case handling - like we probably should reset all the step information when user navigates back and changes some details (but only in some cases!).

Reply:
We have a bunch of multipage flows, and they use one action per page (and one url per page). It might be that we pushed that pattern quite far, and that it would become cumbersome to use for this feature. 

But this feature would need to be a large PR (if we want a single PR for logically cohesive feature) in any case - loads of new content, loads of new logic, loads of tests. I imagine easily over 500 lines. This level of complexity is exactly something that we don't want to complicate further, by adding new and complicated systems to our setup.

So I guess my definition of `as simple as possible` = `as little new cognitive load as possible`. I kind of see introducing the generic multipage form as a refactor of the old setup with one action and one route per page. And refactors are great on small PRs, and not great on large PRs :/ The bigger the PR the scarier it is to include a complex refactor in it.